tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -622,6 +655,32 @@ def test_llmobs_embedding_documents(
             )
         )
 
+    def test_llmobs_similarity_search(self, langchain_openai, langchain_pinecone, mock_llmobs_span_writer, mock_tracer):
+        if langchain_pinecone is None:
+            pytest.skip("langchain_pinecone not installed which is required for this test.")
+        embedding = langchain_openai.OpenAIEmbeddings(model="text-embedding-ada-002")
+        index_name = "langchain-retrieval"
+        vectorstore = langchain_pinecone.PineconeVectorStore(
+            index_name=index_name,
+            embedding=embedding,
+        )
+        cassette_name = (
+            "openai_pinecone_similarity_search_39.yaml" if PY39 else "openai_pinecone_similarity_search.yaml"
+        )
```

Comment:
We don't have python 3.9 specific cassettes for this test case, so you can just use `"openai_pinecone_similarity_search.yaml"` here (no need to special case python 3.9)